models/wrf/shell_scripts/driver.csh

```diff
@@ -372,6 +372,13 @@ while ( 1 == 1 )
 
    echo "ready to integrate ensemble members"
 
+   # Removing old start_member and done_member diagnostics
```

Comment:
If first time through the driver.csh, there could be lingering start_member or done_member files from the prior init_ensemble_var.csh run.  Need to get rid of these or the driver.csh script starts transferring files that do not exist because WRF integration is not yet completed.